packages/suite/src/types/wallet/coinmarketExchangeForm.ts

```diff
@@ -5,7 +5,12 @@ import { FeeLevel } from 'trezor-connect';
 import { ExchangeTrade, ExchangeTradeQuoteRequest, ExchangeCoinInfo } from 'invity-api';
 import { CoinmarketExchangeAction, ExchangeInfo } from '@wallet-actions/coinmarketExchangeActions';
 import { TypedValidationRules } from './form';
-import { FeeInfo, PrecomposedTransactionFinal } from '@wallet-types/sendForm';
+import { FeeInfo, FormState, PrecomposedLevels } from '@wallet-types/sendForm';
+
+export const cryptoInput = 'outputs[0].amount';
```

Comment:
maybe it would be better to have them in uppercase?
it took mi a while to find them here while i was looking at hook code

Reply:
Done